integration-tests/src/test/java/tachyon/LocalTachyonClusterResource.java

```diff
@@ -104,6 +108,10 @@ public LocalTachyonClusterResource(long workerCapacityBytes, int quotaUnitBytes,
     mConfParams = confParams;
   }
 
+  public LocalTachyonClusterResource() {
```

Comment:
could you add a javadoc on this, saying this will construct a local cluster with certain default properties?